main/test/ca/uwaterloo/flix/library/TestDebug.flix

```diff
@@ -172,23 +172,23 @@ mod TestDebug {
     def testFunctionStringify01(): Bool = {
         def f(x: Int32): Int32 = x;
         let s = stringify(f);
-        s == "Int32 -> Int32"
+        s == "Int32 -> Obj"
```

Comment:
I really wonder if this debugging stuff is worth it. But I guess that is a separate discussion.